opal/group.go

```diff
@@ -144,6 +145,42 @@ func resourceGroup() *schema.Resource {
 					},
 				},
 			},
+			"reviewer_stage": {
+				Description: "A reviewer stage for this group. If none are specified, then the admin owner will be used",
```

Comment:
for my knowledge, where is this ("then the admin owner will be used") implemented, or is this comment out-of-date? The validation function (https://github.com/opalsecurity/terraform-provider-opal/pull/12/commits/49d913589b3abea6baf3aa5d05f58364a23c9ae6#diff-b19ea37fd98bc19bb721e1a75ad8b53985cf14d936ffb742c4ad814ad1cc5d9eR39) seemed like it would not allow this

Reply:
Confirming: 
```tf
╷
│ Error: Invalid reviewer configuration. Please specify at least 1 reviewer stage, or set auto_approval to true or set is_requestable to false
│
│   with opal_group.test_group,
│   on provider.tf line 58, in resource "opal_group" "test_group":
│   58: resource "opal_group" "test_group" {
│```

tbf, in Terraform, I would avoid making smart defaults, rather forcing users to set a proper configuration from the start

Reply:
yeah, this is copy pasta from the old reviewer behavior. Will update the comment.